### ISSUE TITLE
Network errors on CI

### ISSUE THREAD

**Issue by kuisathaverat** [elastic/apm-agent-go#630]
the following error happened twice today

```
[2019-09-04T07:39:27.213Z] go: finding gopkg.in/src-d/go-git-fixtures.v3 v3.3.0
[2019-09-04T07:39:31.419Z] verifying go.elastic.co/fastjson@v1.0.0/go.mod: go.elastic.co/fastjson@v1.0.0/go.mod: Get http://proxy.invalid/sumdb/sum.golang.org/supported: dial tcp: lookup proxy.invalid on 169.254.169.254:53: no such host
[2019-09-04T07:39:31.419Z] 2019/09/04 07:39:30 exit status 1
[2019-09-04T07:39:31.419Z] exit status 1
[2019-09-04T07:39:31.419Z] Makefile:29: recipe for target 'check-modules' failed
[2019-09-04T07:39:31.419Z] make: *** [check-modules] Error 1
script returned exit code 2
```

**Comment by kuisathaverat** [comment]
Could be the same in https://github.com/elastic/apm-server/issues/2673

**Comment by kuisathaverat** [comment]
it is a different issue, it could be related to the Go version, failed stages are using 1.13

**Comment by kuisathaverat** [comment]
testing it locally I got a different error

```
verifying github.com/grpc-ecosystem/go-grpc-middleware@v1.0.0: checksum mismatch
	downloaded: h1:BWIsLfhgKhV5g/oF34aRjniBHLTZe5DNekSjbAjIS6c=
	go.sum:     h1:Iju5GlWwrvL6UBg4zJJt3btmonfrMlCDdsejg4CZE7c=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```

**Comment by axw** [comment]
@kuisathaverat this was fixed by #629

**Comment by axw** [comment]
(At least I'm pretty sure it was. Please reopen if it not.)
